dojo/api_v2/serializers.py

```diff
@@ -1788,6 +1788,9 @@ class FindingNoteSerializer(serializers.Serializer):
 
 
 class NotificationsSerializer(serializers.ModelSerializer):
+
+    DEFAULT_NOTIFICATION = NOTIFICATION_CHOICES[-1]
```

Comment:
This is not necessarily stable, if another target for notifications would be introduced. Can `alert` be addressed explicitly?

Reply:
This is not stable in any case, because if you look at the HTML template, order is hardcoded :(

Reply:
Initializing the default with

```suggestion
    DEFAULT_NOTIFICATION = ("alert", "alert")
```

would make the intention clear and the code easier to read. And it's more consistent with the model, where it is like `product_type_added = MultiSelectField(choices=NOTIFICATION_CHOICES, default='alert', blank=True)`

Reply:
Makes sense, but then it should be in Notifications class, so default values are in the single place